src/processes.c

```diff
@@ -775,6 +780,14 @@ static int ps_init(void) {
   pagesize_g = sysconf(_SC_PAGESIZE);
   DEBUG("pagesize_g = %li; CONFIG_HZ = %i;", pagesize_g, CONFIG_HZ);
 
+  // smaps_rollup requires kernel >= 4.14
+  struct stat sb;
+  if (stat("/proc/1/smaps_rollup", &sb) == 0) {
```

Comment:
You could add link to doc to the comment:
https://www.kernel.org/doc/Documentation/ABI/testing/procfs-smaps_rollup

And skip dummy variable use by using `access()` instead of `stat()`.